pt_miniscreen/pages/root/projects/enums.py

```diff
@@ -0,0 +1,16 @@
+from enum import auto, Enum
+
+
+class ProjectState(Enum):
+    IDLE = auto()
+    STARTING = auto()
+    RUNNING = auto()
+    PROJECT_USES_MINISCREEN = auto()
```

Comment:
PROJECT_USES_MINISCREEN doesn't sound like a state to me - or at least it's not mutually exclusive with the others